monitor/src/main/webapp/WEB-INF/tiles/skeleton.jsp

```diff
@@ -41,6 +41,7 @@
     <script src="//ajax.googleapis.com/ajax/libs/jquery/1.11.3/jquery.min.js"></script>
  	<script src="<c:url value="/resources/bootstrap/js/bootstrap.min.js" />"></script>
 	<script src="<c:url value="/resources/bootstrap/js/main.js" />"></script>
+	<script src="<c:url value="/resources/monitor/reload.js" />"></script>
```

Comment:
since this is very specific to the apps list page and especially since it's polling the server, this should not be included in the skeleton. For all other pages than the list page, this doesn't not need to be here.